common-util/Contracts/addresses.jsx

```diff
@@ -123,6 +143,8 @@ export const FALLBACK_HANDLER = {
   10200: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
   80001: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
   31337: '0x0000000000000000000000000000000000000000',
+  42161: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
```

Comment:
There is nothing at this address: https://arbiscan.io/address/0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4

Reply:
no issue here